code/modules/jobs/job_types/roguetown/inquisition/confessor.dm

```diff
@@ -3,8 +3,8 @@ This confessor is different to 'confessors', the people who are hit with the con
 Please do not confuse the two.
 */
 /datum/job/roguetown/shepherd
-	title = "Confessor"
-	flag = CONFESSOR
+	title = "Repentant"
```

Comment:
I have no real thoughts on the name changes for confessor. They're effectively the same, so I'd be fine with having it changed.